eng/pipelines/templates/steps/create-apireview.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  Artifacts: []
+
+steps:
+  - task: DownloadPipelineArtifact@2
+    displayName: Download Packages
+    inputs:
+      artifact: 'packages'
+      path: $(Pipeline.Workspace)/apireviewstaging
+
+  - ${{ each artifact in parameters.Artifacts }}:
+    - task: Powershell@2
+      inputs:
+        filePath: $(Build.SourcesDirectory)/eng/scripts/Create-APIReview.ps1
+        arguments: >
+          -ArtifactPath $(Pipeline.Workspace)/apireviewstaging
```

Comment:
For .NET we actually have each package isolated into its own folder so you should pass the exact path. Similar to places like https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/stages/archetype-net-release.yml#L257 we should add `${{ artifact.Name }}` in the path.